### PR TITLE
Update minimal_Sage_standard.json

### PR DESCRIPTION
**Pull request by amapeters**
brain regions

### CODE REVIEW

File: minimal_Sage_standard.json [hunk 1]
```diff
@@ -86,7 +86,8 @@
         "Schizophrenia",
         "Autism Spectrum Disorder",
         "Control",
-        "Not Applicable"
```
Comment (kdaily) [inline minimal_Sage_standard.json]: defer after discussing #50

File: minimal_Sage_standard.json [hunk 2]
```diff
@@ -110,7 +111,8 @@
         "SH-SY5Y",
         "GLUtamatergic neurons",
         "NeuN+",
-        "NeuN-"
+        "NeuN-",
+        "nan"
```
Comment (kdaily) [inline minimal_Sage_standard.json]: #50

File: minimal_Sage_standard.json [hunk 3]
```diff
@@ -130,8 +132,43 @@
         "dorsolateral prefrontal cortex",
         "anterior cingulate cortex",
         "temporal cortex",
-        "serum",
-        "Not Applicable"
```
Comment (kdaily) [inline minimal_Sage_standard.json]: OK except for removing Not Applicable

File: minimal_Sage_standard.json [hunk 4]
```diff
@@ -130,8 +132,43 @@
...
+        "orbitofrontal cortex",
```
Comment (kdaily) [inline minimal_Sage_standard.json]: see if these match MeSH terms

File: minimal_Sage_standard.json [hunk 5]
```diff
@@ -130,8 +132,43 @@
...
+        "hippocampus",
+        "amygdaloid complex",
+        "striatum",
+        "mediodorsal nucleus of thalamus",
+        "cerebellar cortex",
+        "cerebellum",
+        "serum"
+        
+    ],
+    "BrodmannArea": [
```
Comment (kdaily) [inline minimal_Sage_standard.json]: goes to neuro, if they are integers don't quote them

File: minimal_Sage_standard.json [hunk 6]
```diff
@@ -130,8 +132,43 @@
...
+        "46",
+        "9"
+        
+    ],
+    "hemisphere": [
```
Comment (kdaily) [inline minimal_Sage_standard.json]: remove from this PR, make a new one

File: minimal_Sage_standard.json [hunk 7]
```diff
@@ -130,8 +132,43 @@
...
+        "Left",
+        "Right"
+    ],
+    "RIN": [
+        
+    ],
+    "PMI": [
```
Comment (kdaily) [inline minimal_Sage_standard.json]: move to neuro

File: minimal_Sage_standard.json [hunk 8]
```diff
@@ -130,8 +132,43 @@
...
+        
+   ],
+    "pH": [
```
Comment (kdaily) [inline minimal_Sage_standard.json]: move to neuro

File: minimal_Sage_standard.json [hunk 9]
```diff
@@ -146,8 +183,7 @@
         "Synodos",
         "BSMN",
         "CSBC",
-        "PSON",
-        "Not Applicable"
```
Comment (kdaily) [inline minimal_Sage_standard.json]: keep Not App

File: minimal_Sage_standard.json [hunk 10]
```diff
@@ -130,8 +132,43 @@
         "dorsolateral prefrontal cortex",
         "anterior cingulate cortex",
         "temporal cortex",
-        "serum",
-        "Not Applicable"
+        "orbitofrontal cortex",
+        "ventrolateral prefrontal cortex",
```
Comment (amapeters) [inline minimal_Sage_standard.json]: no mesh term

File: minimal_Sage_standard.json [hunk 11]
```diff
@@ -130,8 +132,43 @@
...
+        "medial frontal cortex",
```
Comment (amapeters) [inline minimal_Sage_standard.json]: no exact mesh term

File: minimal_Sage_standard.json [hunk 12]
```diff
@@ -130,8 +132,43 @@
...
+        "primary motor cortex",
+        "primary somatosensory cortex",
+        "posteroinferior parietal cortex",
+        "primary auditory cortex",
+        "posterior superior temporal cortex",
+        "inferolateral temporal cortex",
```
Comment (amapeters) [inline minimal_Sage_standard.json]: no exact mesh term

File: minimal_Sage_standard.json [hunk 13]
```diff
@@ -130,8 +132,43 @@
...
+        "primary visual cortex",
+        "hippocampus",
+        "amygdaloid complex",
```
Comment (amapeters) [inline minimal_Sage_standard.json]: no exact mesh term

File: minimal_Sage_standard.json [hunk 14]
```diff
@@ -130,8 +132,43 @@
...
+        "striatum",
```
Comment (amapeters) [inline minimal_Sage_standard.json]: no exact mesh term

File: minimal_Sage_standard.json [hunk 15]
```diff
@@ -130,8 +132,43 @@
...
+        "mediodorsal nucleus of thalamus",
```
Comment (amapeters) [inline minimal_Sage_standard.json]: no exact mesh term

File: minimal_Sage_standard.json (hunk 5 above)
Comment (amapeters) [inline minimal_Sage_standard.json]: moved to neuro

File: minimal_Sage_standard.json (hunk 6 above)
Comment (amapeters) [inline minimal_Sage_standard.json]: moved to neuro

File: minimal_Sage_standard.json (hunk 8 above)
Comment (amapeters) [inline minimal_Sage_standard.json]: moved to neuro

File: minimal_Sage_standard.json (hunk 13 above)
Comment (kdaily) [inline minimal_Sage_standard.json]: http://purl.obolibrary.org/obo/UBERON_0001876 (amygdaloid complex is a synonym for amygdala)

File: minimal_Sage_standard.json (hunk 15 above)
Comment (kdaily) [inline minimal_Sage_standard.json]: http://purl.obolibrary.org/obo/UBERON_0002739

File: minimal_Sage_standard.json (hunk 14 above)
Comment (kdaily) [inline minimal_Sage_standard.json]: http://purl.obolibrary.org/obo/UBERON_0002435

**Pull request merged**